.circleci/config.yml

```diff
@@ -294,10 +305,17 @@ commands:
               --param AMP_ZYNC_IMAGE=quay.io/3scale/zync:nightly \
               --param AMP_APICAST_IMAGE=quay.io/3scale/apicast:nightly \
               --param AMP_SYSTEM_IMAGE=quay.io/3scale/porta:nightly \
+              --param MEMCACHED_IMAGE=registry.redhat.io/3scale-amp2/memcached-rhel7:3scale2.9 \
```

Comment:
should we extract the `2.9` in some param / global var, to make it a little more prominent (so we don't forget to update it? when 2.10 is ga)

...not that it really matters for this particular image though... still ok if you don't want to change.

Reply:
Updating it. Makes all the sense for the master branch. Less important for this stable branch, but still I will implement it